public/js/load_comment.js

```diff
@@ -0,0 +1,19 @@
+const addComment = (data)=>{
+  const comment = document.createElement('div');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/soutenance14/ST/pullRequest?prid=8161867)
```suggestion
  const comment = document.createElement("div");
```